documentation/markup.rst

```diff
@@ -246,14 +246,16 @@ directives.)  Looking at this example,
 
 ::
 
-   .. function:: foo(x)
-                 foo(y, z)
+   .. function:: remove(path, *, dir_fd=None)
+                 unlink(path, *, dir_fd=None)
       :bar: no
```

Comment:
Leaving this after updating the example looks a bit out of place though.  Are there any real-world examples that have options?
Otherwise why not just leave the original example unchanged?

Reply:
Yeah, let's keep the orignal example.